translations/de.json

```diff
@@ -1,13 +1,13 @@
 {
-  "eventDetails": "Veranstaltungsdetails"
-  "speakers": "Vortragende"
-  "talks": "Vorträge"
-  "recordings": "Aufzeichnungen"
-  "faq": "FAQ/häufig gestellte Fragen"
-  "saveDate": "Folgende Sitzung speichern: {date}"
-  "signUp": "Einloggen für Updates"
-  "joinSlack": "Trete der Slack Gruppe bei"
-  "showDatesEDT": "zeige Uhrzeiten in Zeitzone EDT an"
+  "eventDetails": "Veranstaltungsdetails",
```

Comment:
thanks for catching this - was so busy catching up on PRs my reviews are slipping! that would have been problematic down the road, so thanks for fixing!